app/screens/route-details/components/start-ride-button.tsx

```diff
@@ -0,0 +1,95 @@
+import { Alert, Dimensions, Image, ImageStyle, Platform, View, ViewStyle } from "react-native"
+import { observer } from "mobx-react-lite"
+import { useSafeAreaInsets } from "react-native-safe-area-context"
+import HapticFeedback from "react-native-haptic-feedback"
+import { Button } from "../../../components"
+import { isRTL, translate } from "../../../i18n"
+import { RouteItem } from "../../../services/api"
+import { differenceInMinutes, isAfter } from "date-fns"
+import { timezoneCorrection } from "../../../utils/helpers/date-helpers"
+import { color } from "../../../theme"
+import { useStores } from "../../../models"
+
+const { width: deviceWidth } = Dimensions.get("screen")
+
+const START_RIDE_BUTTON: ViewStyle = {
+  shadowOffset: { width: 0, height: 2 },
+  shadowOpacity: 0.2,
+  shadowRadius: 2,
+  shadowColor: "#333",
+  elevation: 4,
+}
+
+const TRAIN_ICON: ImageStyle = {
+  width: 22.5,
+  height: 22.5,
+  tintColor: "white",
+  transform: [{ rotateY: isRTL ? "180deg" : "0deg" }],
+}
+
+interface StartRideButtonProps {
+  route: RouteItem
+  screenName: "routeDetails" | "activeRide"
+}
+
+export const StartRideButton = observer(function StartRideButton(props: StartRideButtonProps) {
```

Comment:
Don't animate start ride button when the view appears. Do it only when the user starts/end the ride.